pkgs/development/python-modules/nettigo-air-monitor/default.nix

```
@@ -14,7 +14,7 @@
 
 buildPythonPackage rec {
   pname = "nettigo-air-monitor";
-  version = "2.2.0";
+  version = "2.2.1";
   format = "setuptools";
 
   disabled = pythonOlder "3.8";
```

```suggestion
  disabled = pythonOlder "3.10";
```
https://github.com/bieniu/nettigo-air-monitor/blob/727b9f2f23b81de266534db360c94b49afde4134/setup.py#L24